containers/PollPR.py

```diff
@@ -0,0 +1,681 @@
+"""
+# Aiming to be used with Hydra (Nixos) builder for Ghaf project https://github.com/tiiuae/ghaf
+
+# Activates Hydra (Ghaf docker based Hydra build system implementation) to build new open Pull Requests (for the main repo under observations)
+# Rebuilds still open (and previously built) PR if there has been new changes for that PR
+# Uses Ghaf build tools from https://github.com/tiiuae/ci-public
+
+# Can be used in service mode (polling frequently changes) or cherrypicking some wished open PR 
+# Keeps internal recoed which open PRs or changed PRs have been allready processed (given Hydra build commands)
+#
+#
+#
+# Author mika.nokka1@gmail.com 13.6.2023
+"""
+
+# pragma pylint: disable=bad-indentation,superfluous-parens,line-too-long,global-statement,too-many-arguments,consider-using-with,no-else-break,no-else-return,unspecified-encoding,unused-import,too-many-locals,too-many-branches,too-many-statements,unused-argument,no-member,unsubscriptable-object,pointless-statement
```

Comment:
Yeah, this is not the way ignores are supposed to be used.
Ignore particular warning when you are absolutely sure that your code is the right way to go, and explain yourself in the comments: "Ignoring warning xyz because..."

Reply:
re organized pylint skipping usage, fixed most of the pylint complains which were reasonable nor features in its workings